### PR TITLE
Fix duplicate error messages issue

### LINKED ISSUE #29: Errors are reported twice
It seems that currently any error is reported twice:

```
  22 passing (13s)
  1 failing

  1) eslint should have no errors in lib:
     Code did not pass lint rules
/Users/tbn/Code/ember-cli/lib/utilities/reexport-template.js
  2:1  error  Use the global form of "use strict"  strict
  3:3  error  Use the global form of "use strict"  strict

✖ 2 problems (2 errors, 0 warnings)

  Error: Code did not pass lint rules
  lib/utilities/reexport-template.js
    2:1  error  Use the global form of "use strict"  strict
    3:3  error  Use the global form of "use strict"  strict

  ✖ 2 problems (2 errors, 0 warnings)

      at Context.<anonymous> (node_modules/mocha-eslint/index.js:27:13)
```

Interesting about this is that one of the outputs is using relative paths, while the other is using the absolute path of the file.

### PR DESCRIPTION
**Pull request by Turbo87**
This PR resolves #29

### CODE REVIEW

**Comment by IanVS** [comment]
Awesome work tracking this down and making a fix!  I saw that mocha has a `--full-trace` option.  I wonder if that would also solve your issues.  

Regardless, even though this is a workaround as you said, I'm fine with merging this since it's not practical to force everyone to use that mocha option (if it even works).  Looks like you've fixed the tests.  Good to go on your end?

File: index.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ function test(p, opts) {
     ) {
       throw new Error(
         chalk.red('Code did not pass lint rules') +
-        formatter(report.results)
+        formatter(report.results).replace(process.cwd() + '/', '')
```
Comment (IanVS) [inline index.js]: Actually, do you think you could add a small comment to explain why the replace is needed?  I can see this causing some head-scratching in the future.

Comment (Turbo87) [inline index.js]: sure, done

**Comment by Turbo87** [comment]
> Good to go on your end?

yes, I think this should solve the issue. might be good if you could verify that everything else still works properly though.

**Comment by IanVS** [comment]
Thanks!

**Pull request merged**